src/vs/workbench/common/editor/editorGroupModel.ts

```diff
@@ -220,9 +220,9 @@ export class EditorGroupModel extends Disposable implements IEditorGroupModel {
 
 	private locked = false;
 
-	private selection: EditorInput[] = [];					// editors in selected state, first one is active
+	private selection: EditorInput[] = [];					// editors in selected state, first one is active TODO align this with transient editors and use a Set<EditorInput> instead
 
-	private set active(editor: EditorInput | null) {
+	private set active(editor: EditorInput | null) { // TODO: this misses an event when selection changes and should be a method?
```

Comment:
@benibenj this is dangerous because we might change the `selection` without emitting an event

Reply:
It is always possible to forget to trigger an event. Same with activeEditor before. But what I can do is remove the setter, this would make sure changing active editor has to go over the selection, making the dev aware of it. Would this be better?